src/lib/json-rpc-router.ts

```diff
@@ -9,8 +9,8 @@ export type RequestContext = {
   transport: 'http' | 'ws';
   dwn: Dwn;
   socketConnection?: SocketConnection;
-  /** The `MessageSubscriptionHandler` associated with a subscription request, only used in `ws` requests */
-  subscriptionHandler?: MessageSubscriptionHandler;
+  /** The `GenericMessage` handler associated with a subscription request, only used in `ws` requests */
+  subscriptionHandler?: (message: GenericMessage) => void;
```

Comment:
Also for my learning: what's the reason behind not using the declared type directly?

Reply:
Fixed